Library/Formula/gauge.rb

```diff
@@ -76,6 +76,11 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    Installing gauge with homebrew doesn't install Gauge plugins which are required to run Gauge specs.
+    To install Gauge plugin, please explicitly run `gauge --install <plugin_name>`. Language plugins supported are `java`, `csharp` and `ruby`. Report plugins available are `html-report` and `xml-report`.
```

Comment:
If we *have* to have these caveats they should be a lot shorter, or at least broken onto multiple lines.

I'm not sure why we couldn't have options like `without-java-plugin` and then in `postinstall` do something like `gauge --install java if build.with? "java-plugin"` or such, but either way, the caveats need to be peeled back a bit.